src/main/java/org/folio/repository/CustomFieldsRepositoryImpl.java

```diff
@@ -36,12 +39,13 @@
    * @param tenantId - tenant id
    */
   @Override
-  public Future<Optional<CustomField>> save(CustomField customField, String tenantId) {
+  public Future<CustomField> save(CustomField customField, String tenantId) {
     Future<String> future = Future.future();
     logger.debug("Saving a custom field with id: {}.", customField.getId());
     PostgresClient.getInstance(vertx, tenantId)
       .save(CUSTOM_FIELDS_TABLE,  customField.getId(), customField, future);
-    return future.compose(customFieldId -> findById(customFieldId, tenantId));
+
+    return future.map(customField);
```

Comment:
suggest to add ext translator here also..
in general, it should be applied to all repository methods, but finders can be ignored for now

Reply:
@dmtkachenko Thanks, added